oak/server/rust/oak_loader/src/options.rs

```diff
@@ -80,6 +87,20 @@ pub struct Opt {
     config_files: Vec<ConfigEntry>,
 }
 
+#[derive(Debug, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+struct SignatureManifest {
+    name: String,
+    signatures: HashMap<String, Signature>,
+}
+
+#[derive(Debug, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+struct Signature {
```

Comment:
How is this different from SignatureConfiguration ? could you add comments to clarify?